### ISSUE TITLE
OptiTrackCore cocoapod issue on XCode10

### ISSUE THREAD

**Issue by alexbrie** [optimove-tech/iOS-SDK-Integration-Guide#7]
TL;DR: the XCode 10 improved parallel build system doesn't work with Optimove cocoapod dependency. A temporary fix is to switch to the legacy XCode build system, but this has obvious downsides performance wise. 

Problem:

When building a project that uses the Optimove cocoapod(and its OptiTrackCore dependency), on XCode 10 with the new (parallelised) build system, building fails due to the bad configuration of the OptiTrackCore Info.plist file - see the attached image

<img width="496" alt="screen shot 2018-09-17 at 12 26 52" src="https://user-images.githubusercontent.com/17677/45615634-cc84df00-ba75-11e8-9c12-1aacca94ca9d.png">

If I remove it from the target completely, I get the app installation error 

<img width="438" alt="screen shot 2018-09-17 at 12 31 42" src="https://user-images.githubusercontent.com/17677/45615698-f2aa7f00-ba75-11e8-9263-543cac873096.png">

**Comment by eligoptimove** [comment]
Hello @alexbrie. Is the issue still there? Try to update to the latest version 2.1.2
